src/invoices/InvoiceDetails/InvoiceDetails.js

```diff
@@ -62,6 +64,8 @@ class InvoiceDetails extends Component {
     );
   }
 
+  onInvoiceLinesLoaded = (invoiceLineTotal = 0) => this.setState({ invoiceLineTotal });
```

Comment:
I don't see any reason to store this value. You could get it from resource directly.

Reply:
I did this because we have no data on the number of lines in this component. Another solution is to connect the InvoiceDetails component to the strips store.

Reply:
right, please use resource from above